packages/e2e-tests/specs/editor/various/block-deletion.test.js

```diff
@@ -192,6 +192,7 @@ describe( 'deleting all blocks', () => {
 		// Add and remove a block.
 		await insertBlock( 'Image' );
 		await page.waitForSelector( 'figure[data-type="core/image"]' );
+		await page.keyboard.press( 'ArrowUp' );
```

Comment:
@tellthemachines This didn't change though, right? If contains cursor option is selected, Up Arrow on latest trunk should not focus the wrapper either.

Reply:
Sure, but with these changes the wrapper won't be focused after insertion anymore. This is only relevant when you add a block, immediately realise it's the wrong block, and want to quickly delete it. It would be nice to preserve that ability, but we should be able to find a workaround for it.

Reply:
> This is only relevant when you add a block, immediately realise it's the wrong block, and want to quickly delete it. It would be nice to preserve that ability, but we should be able to find a workaround for it.

I wonder if in a follow-up, we can look at in placeholder states that don't have an editable input field (e.g. the Columns placeholder state, but not the Twitter embed input field), and see if we can get backspace to still delete the block?